tests/integration_tests/with_db/test_upload_model.py

```diff
@@ -48,11 +60,7 @@ def test_model_upload(download_sample_models: Path, bucketfs_location: BucketFSL
         runner = CliRunner()
         result = runner.invoke(upload_model.main, args_list)
         assert result.exit_code == 0
-
-        downloaded_files = set(
-            i.name for i in download_path.iterdir() if i.is_file())
-        uploaded_files = set(
-            i for i in bucketfs_location.list_files_in_bucketfs(upload_path))
-        assert uploaded_files == downloaded_files
+        assert str(upload_path.with_suffix(".tar.gz")) in bucketfs_location.list_files_in_bucketfs(".")
     finally:
-        postprocessing.cleanup_buckets(bucketfs_location, upload_path)
+        pass
```

Comment:
is the pass really needed here?

Reply:
the question is actually, can we enable the postprocessing.cleanup again

Reply:
```suggestion
```